SciPy2024/module2/embeddings.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>@lsetiawan: The dimensionality of the embedding model is mentioned here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-ssec/tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>